week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -45,17 +45,146 @@ Very doubtful.
 
 // This should log "The ball has shaken!"
 // and return the answer.
-function shakeBall() {}
+
+//Created a switch 
```

Comment:
This 100% works, but you may have felt as you were writing it that it was a little repetitive (you needed to write down every string twice, which took quite a lot of typing!)

In week three we're going to learn more about arrays and objects which would help us write this with less code - give some thought as to how you may want rewrite this to be smaller, as we learn those things next week!